ot-node.js

```diff
@@ -240,7 +242,6 @@ class OTNode {
             }
         }
 
-        Object.seal(config);
```

Comment:
I'm worried it might not be safe to seal the config after loading the modules, is there a way we could check this? Maybe we can log the  config before  and after the loading

Reply:
Object.seal was moved before we introduced ha node sync using health check route. I'll return it here.